docs/administration/upgrade/to_v6.0.adoc

```diff
@@ -1,12 +1,33 @@
-= Upgrading to Logging v6.0
+= What's New In Logging 6.0
 
 == Overview
-Logging v6.0 is a major change from earlier releases and is the realization of several longstanding goals of Cluster Logging:
+Logging version 6.0 is a major change from earlier releases and is the realization of several longstanding goals.
 
-* Distinct operators to support logging components (e.g. collectors, storage, visualization)
-* Remove support of managed log storage and visualization based upon the Elastic products (i.e. Elasticsearch, Kibana)
-* Remove support of Fluentd log collector implementation
-* Remove support of the ClusterLogging.logging.openshift.io and ClusterLogForwarder.logging.openshift.io
+The main highlights of these changes are:
+
+* There is now distinct operators to support separate logging components (e.g. collectors, storage, visualization)
+* Cluster Logging Operator no longer implements managed log storage and visualization based upon the Elastic products (i.e. Elasticsearch, Kibana)
```

Comment:
The original statement was about "Cluster Logging" and this change is more specific to the actual operator.  If we leave as "CLO" then I think it would better be described like:

The ClusterLoggingOperator no longer manages log storage or visualization of any kind including LokiStack resource or Elastic products (i.e. Elasticsearch, Kibana)